app/controllers/admins_controller.rb

```diff
@@ -475,6 +475,8 @@ def detectAndConvertRoster(roster)
       # First Name(6), (skip)(7), Andrew ID(8), (skip)(9), School(10), 
       # Major(11), Year(12), (skip)(13), Grade Policy(14), ...
 
+      # Sanitize roster input, ignoring empty / incomplete lines.
+      parsedRoster.select! { |row| row.length == CMU_ROSTER_COLUMNS }
```

Comment:
Earlier on, we check that the first element in parsedRoster is this length too.  Can't we merge these two statements?  (Also, maybe we should rename the constant to ROSTER_COLUMNS.  It's based off of CMU, but there's no reason other schools couldn't prepare data in the same format)